lib/glific/flows/case.ex

```diff
@@ -84,8 +84,30 @@ defmodule Glific.Flows.Case do
   Validate a case
   """
   @spec validate(Case.t(), Keyword.t(), map()) :: Keyword.t()
-  def validate(_case, errors, _flow) do
-    errors
+  def validate(cases, errors, flow) do
+    wait_for_response_words =
+      cases.arguments
+      |> List.first()
+      |> String.split(", ")
+      |> Enum.reduce(MapSet.new(), &MapSet.put(&2, &1))
+      |> MapSet.delete(nil)
+
+    flow_keywords =
+      Glific.Flows.flow_keywords_map(flow.organization_id)["published"]
+      |> Map.keys()
+      |> Enum.reduce(MapSet.new(), &MapSet.put(&2, &1))
+
+    if MapSet.disjoint?(flow_keywords, wait_for_response_words) do
+      errors
+    else
+      used_flow_keywords = MapSet.intersection(flow_keywords, wait_for_response_words)
+
+      Enum.reduce(
+        used_flow_keywords,
+        errors,
+        &(&2 ++ [flowContext: "\"#{&1}\" has already been used as a keyword for a flow"])
```

Comment:
is there a possibility to add the flow name also in this message?

Reply:
@pankaj-ag dont see a use for that. will check with @abhi1203

Reply:
Yeah that would be even better. if we can say `"Start" has already been used as a keyword for flow "flowname"` thanks for spotting